### ISSUE TITLE
Psycopg2 dependency changes its name.

### ISSUE THREAD

**Issue by almereyda** [modoboa/modoboa-installer#185]
# Current behaviour

During installation

```
/srv/modoboa/env/local/lib/python2.7/site-packages/psycopg2/__init__.py:144: UserWarning: The psycopg2 wheel pac
kage will be renamed from release 2.8; in order to keep installing from binary please use "pip install psycopg2-
binary" instead. For details see: <http://initd.org/psycopg/docs/install.html#binary-install-from-pypi>.
  """)
```

and in messages to `root` / `vmail` / `postmaster`

```
Return-Path: <vmail@email.ecobytes.net>
Delivered-To: <postmaster@email.ecobytes.net>
Received: from email.ecobytes.net
	by email.ecobytes.net (Dovecot) with LMTP id JKqPGCDwjVr2KAAAsI18iQ
	for <postmaster@email.ecobytes.net>; Wed, 21 Feb 2018 22:28:02 +0000
Received: by email.ecobytes.net (Postfix, from userid 1002)
	id 8BCEDDF496; Wed, 21 Feb 2018 22:28:02 +0000 (GMT)
From: root@email.ecobytes.net (Cron Daemon)
To: vmail@email.ecobytes.net
Subject: Cron <vmail@email>   $PYTHON $INSTANCE/manage.py handle_mailbox_operations
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit
X-Cron-Env: <PYTHON=/srv/modoboa/env/bin/python>
X-Cron-Env: <INSTANCE=/srv/modoboa/instance>
X-Cron-Env: <SHELL=/bin/sh>
X-Cron-Env: <HOME=/srv/vmail>
X-Cron-Env: <PATH=/usr/bin:/bin>
X-Cron-Env: <LOGNAME=vmail>
Message-Id: <20180221222802.8BCEDDF496@email.ecobytes.net>
Date: Wed, 21 Feb 2018 22:28:02 +0000 (GMT)

/srv/modoboa/env/local/lib/python2.7/site-packages/psycopg2/__init__.py:144: UserWarning: The psycopg2 wheel package will be renamed from release 2.8; in order to keep installing from binary please use "pip install psycopg2-binary" instead. For details see: <http://initd.org/psycopg/docs/install.html#binary-install-from-pypi>.
  """)
```

Those are sent every minute.

# Expected behavior

The dependency is upgraded and no errors appear.

**Comment by almereyda** [comment]
For now we have applied a dirty fix by appending `2>/dev/null` to the entry for `handle_mailbox_operations` in `/etc/cron.d/modoboa`, in so these messages do not spam the postmaster inbox anymore.

**Comment by y0va** [comment]
Adding `PYTHONWARNINGS=ignore` at the top of `/etc/cron.d/modoboa` switches of the warnings for the other cronjobs too.

**Comment by kryskool** [comment]
Hi

In a python script i have made this code [from Python docs](https://docs.python.org/2/library/warnings.html#temporarily-suppressing-warnings)

```
import warnings

with warnings.catch_warnings():
    warnings.simplefilter("ignore")
    import psycopg2
```

**Comment by fyfe** [comment]
This has been fixed in modoboa/modoboa#1418, it'll be in the next release (1.10.4)
